messaging-activemq/src/main/resources/org/wildfly/extension/messaging/activemq/LocalDescriptions.properties

```diff
@@ -43,6 +43,7 @@ address-setting.slow-consumer-threshold=The minimum rate of message consumption
 address-setting=An address setting defines some attributes that are defined against an address wildcard rather than a specific queue.
 bindings-directory.path=The directory in which the bindings journal lives. The default is ${jboss.server.data.dir}/messagingbindings.
 bridge.add=Operation adding a bridge.
+bridge.call-timeout=Timeout (in milliseconds) for blocking calls.
```

Comment:
Don't specify the unit there as it is already defined in the attribute definition

Reply:
OK...